spec/python-spec.coffee

```diff
@@ -296,3 +296,34 @@ describe "Python grammar", ->
       expect(tokens[1][0]).toEqual value: 'SELECT bar', scopes: ['source.python', scope]
       expect(tokens[2][0]).toEqual value: 'FROM foo', scopes: ['source.python', scope]
       expect(tokens[3][0]).toEqual value: delim, scopes: ['source.python', scope, 'punctuation.definition.string.end.python']
+
+
+  it "tokenizes SQL inline highlighting on blocks with a CTE", ->
```

Comment:
I prefer to have explicit tests since they are technically checking different things. This also makes it easier to catch regressions in the future.